lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -361,16 +361,8 @@ public function getListTableIndexesSQL($table, $currentDatabase = null)
      */
     private function getTableWhereClause($table, $classAlias = 'c', $namespaceAlias = 'n')
     {
-        $whereClause = $namespaceAlias.".nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast') AND ";
-        if (strpos($table, ".") !== false) {
-            list($schema, $table) = explode(".", $table);
-            $schema = "'" . $schema . "'";
-        } else {
-            $schema = "ANY(string_to_array((select replace(replace(setting,'\"\$user\"',user),' ','') from pg_catalog.pg_settings where name = 'search_path'),','))";
-        }
-
-        $table = new Identifier($table);
```

Comment:
I think we'll have to leave this logic in for incoming already quoted identifiers. Example:

```php
// "foo'bar"

$tableIdentifier = new Identifier($table);
$tableName = $table->getName(); // foo'bar
$tableName = $this->quoteStringLiteral($tableName); // 'foo''bar'
```

This should be correct.